src/index.ts

```diff
@@ -30,6 +30,17 @@ function emitDom() {
     const inputFolder = path.join(__SOURCE_DIRECTORY__, "../", "inputfiles");
     const outputFolder = path.join(__SOURCE_DIRECTORY__, "../", "generated");
 
+    // ${name} will be substituted with the name of an interface
+    const removeVerboseIntroductions: (RegExp | [RegExp, string])[] = [
+        [/^(The|A) ${name} interface of (the\s*)*([a-z\s]+ API)(\\\'s)?/, 'An interface of the $3 '],
+        /^(The|A) ${name} (interface|event|object) (is|represents|represent|describes|defines)?/,
+        /^An object implementing the ${name} interface (is|represents|represent|describes|defines)/,
+        /^The ${name} is an interface representing/,
+        /^This type (is|represents|represent|describes|defines)?/,
```

Comment:
what does `represent` match?

Reply:
Looks like XMLDocument and DOMImplementation have typoes in the source.

Reply:
Just fixed those typos on upstream, maybe the json should be updated soon.

Reply:
@mihailik Can `represent` be removed now? XMLDocument and DOMImplementation have no such typo anymore.